franka_gazebo/src/franka_hw_sim.cpp

```diff
@@ -200,7 +205,7 @@ void FrankaHWSim::initFrankaModelHandle(
         " joints were found beneath the <transmission> tag, but 2 are required.");
   }
 
-  for (auto& joint : transmission.joints_) {
+  for (const auto & joint : transmission.joints_) {
```

Comment:
Automatically applied by clang-tidy.

Reply:
There is a space between the auto and the &. Actually this should not pass the CI :thinking:

Reply:
Ah, that's strange.

I also noticed that the [clang-tidy vscode](https://marketplace.visualstudio.com/items?itemName=notskm.clang-tidy) doesn't take that rule into account when formatting. This can of course be a bug with the extensions and not the `.clang-tidy` config. If it is a bug in the extension it will probably be fixed when the C++ extension adds clang-tidy support (see https://github.com/microsoft/vscode-cpptools/issues/2908).